addons/google_calendar/models/google_sync.py

```diff
@@ -173,7 +173,13 @@ def _google_error_handling(self, http_error):
         # We only handle the most problematic errors of sync events.
         if http_error.response.status_code in (403, 400):
             response = http_error.response.json()
-            if self._name == 'calendar.event':
+            if not self.exists():
+                reason = _("Google gave the following explanation: %s", response['error'].get('message'))
+                error_log = "Error while syncing record. It does not exists anymore in the database. %s" % reason
+                _logger.error(error_log)
```

Comment:
It's a bit weird to translate the reason but not the generic error message and log the mix of the two. Even more, since the details provided by google are in English anyway.

Reply:
Also, you need to regenerate the pot for the translated string.

Reply:
I will keep everything in English. it's for server logs anyway.